charts/overprovisioner/values.yaml

```diff
@@ -37,16 +37,17 @@ capacity:
     cpu: 10m
     # -- Memory resource requests and limits for the pause pods.
     memory: 16Mi
+    extra: {}
```

Comment:
This would need a [helm-docs](https://github.com/norwoodj/helm-docs) comment and you'd need to run `helm-docs` locally to update the README.

Reply:
helm-doc added